src/ios/CDVCapture.m

```diff
@@ -292,6 +295,65 @@ - (CDVPluginResult*)processVideo:(NSString*)moviePath forCallbackId:(NSString*)c
     return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:fileArray];
 }
 
+- (void)showAlertIfAccessProhibeted
+{
+    if (![self hasCameraAccess]) {
+        [self showPermissionsAlert];
+    }
+}
+
+- (BOOL)hasCameraAccess
+{
+    if (![AVCaptureDevice respondsToSelector:@selector(authorizationStatusForMediaType:)]) {
+        return YES;
+    }
```

Comment:
I'm ok with that. Actually all iOS <= 10.0 currently are not available for installation on phones. So, it makes sense. 
Initially I tried to keep support line the same as `cordova-plugin-camera` does.

So, just to double check: 
1. I remove all `#pragma` statements
2. I remove all `&UIApplicationOpenSettingsURLString != NULL` checks
3. I add `ios` version restriction to be iOS 8+